packages/adena-extension/src/inject/message/message-handler.ts

```diff
@@ -82,6 +82,9 @@ export class MessageHandler {
       case 'ADD_ESTABLISH':
         HandlerMethod.addEstablish(message, sendResponse);
         break;
+      case 'CHECK_CONNECTION':
+        HandlerMethod.checkEstablished(message, sendResponse);
```

Comment:
Okay, I'll look into this.
btw, how do you test locally the extension before publishing it ? 
Didn't tested this if I remember correctly 😅

Reply:
Aha, We didn't explain enough about running extensions.

When you run the `yarn build` command, it creates a `dist` directory inside the `packages/adena-extension` directory.

You can then register the result of the build command as an extension, and in the case of Chrome, you can load your extension after enabling developer mode at `chrome://extensions` 😄

Reply:
Fixed and tested locally - sending `WALLET_LOCKED` if needed, and `ALREADY_CONNECTED` response on success

Reply:
I think it would be better to define a new success type instead of `ALREADY_CONNECTED`.

`ALREADY_CONNECTED` returns a non-zero code and returns a message that can be confusing to the user.